MCP-Cursor-راهنمای-فارسی.md

```
@@ -0,0 +1,2915 @@
+# "یک پروژه task-manager بساز"
+# "یک API endpoint برای tasks با تمام عملیات CRUD بساز"
+# "کامپوننت React برای نمایش لیست tasks بساز"
+# "Docker compose را راه‌اندازی کن"
+```
+
+### سناریو 2: اتوماسیون DevOps با MCP
+
+#### هدف: اتوماسیون فرآیند CI/CD
+
+**سرور MCP برای DevOps:**
+```python
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)